### PR TITLE
[ZEPPELIN-1260] R interpreter doesn't work with Spark 2.0

### PR DESCRIPTION
**Pull request by Leemoonsoo**
### What is this PR for?
This PR fixes R interpreter doesn't work with Spark 2.0

### What type of PR is it?
Bug Fix

### Todos
* [x] - Create and inject SparkSession into SparkRInterpreter

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1260

### How should this be tested?
Run Zeppelin with Spark 2.0 and run following codes and see if it returns `[1] 3`
```
%r 
localDF <- data.frame(name=c("a", "b", "c"), age=c(19, 23, 18))
df <- createDataFrame(spark, localDF)
count(df)
```

### Screenshots (if appropriate)
image

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: spark/src/main/java/org/apache/zeppelin/spark/ZeppelinR.java [hunk 1]
```diff
@@ -107,9 +108,11 @@ public Object getValue() {
    * @param rCmdPath R repl commandline path
    * @param libPath sparkr library path
    */
-  public ZeppelinR(String rCmdPath, String libPath, int sparkRBackendPort) {
+  public ZeppelinR(String rCmdPath, String libPath,
+                   int sparkRBackendPort, SparkVersion sparkVersion) {
```
Comment (bzz) [inline spark/src/main/java/org/apache/zeppelin/spark/ZeppelinR.java]: This is a nitpick (so no need to change if you do not feel like it) but Is that IDE formatting?
As [horizontal alignment](https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment) is usually discouraged.

**Comment by bzz** [comment]
Looks great to me, modulo question above.

Thank you for prompt fix @Leemoonsoo ! 
Shall this be merged to master or branch-0.6 as well? Asking as JIRA fix version is not set for this one.

**Comment by Leemoonsoo** [comment]
@bzz Thanks for quick review. Updated the indent.
This should be merged to branch-0.6 as well

**Comment by echarles** [comment]
@minahlee Just saw you comment on ZEPPELIN-1260, but @Leemoonsoo has been fast...

I have tested the changes, and now sparkr dataframes are working well (with the `spark` variable which referts to the new Spark2 session).

The R tutorial note refers to sqlContext which is no more present

```
Error in dispatchFunc("createDataFrame(data, schema = NULL, samplingRatio = 1.0)", : object 'sqlContext' not found
```

Patch could also expose sqlContext.

**Comment by Leemoonsoo** [comment]
Thanks @echarles for the review. I have exposed sqlContext as well

**Comment by echarles** [comment]
LGTM

**Comment by Leemoonsoo** [comment]
Merge it into master and branch-0.6 if there're no more discussion

**Pull request closed without merging**